src/XcooBee/Core/Api/System.php

```diff
@@ -198,6 +198,33 @@ public function getEvents($config = [])
 
         return $events;
     }
+
+    /**
+     * trigger test event to campaign webhook
+     *
+     * @param string $type
+     * @param array $config
+     *
+     * @return Response
+     * @throws XcooBeeException
+     */
+    public function triggerEvent($type, $config = [])
+    {
+        $campaignId = $this->_getCampaignId(null, $config);
```

Comment:
For `type` we should check the list of valid events locally before sending to XcooBee backend.